### ISSUE TITLE
problems on macosx

### ISSUE THREAD

**Issue by gWOLF3** [werner-duvaud/muzero-general#87]
not able to finish installing requirements on macosx. 

using python3

```
Building wheels for collected packages: box2d-py
  Building wheel for box2d-py (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/gWOLF3/.asdf/installs/python/3.8.1/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/wl/4_yzp5590b5gxg0wt439nw040000gp/T/pip-install-jhkyzqvc/box2d-py/setup.py'"'"'; __file__='"'"'/private/var/folders/wl/4_yzp5590b5gxg0wt439nw040000gp/T/pip-install-jhkyzqvc/box2d-py/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/wl/4_yzp5590b5gxg0wt439nw040000gp/T/pip-wheel-sm5chbv1
```

**Comment by werner-duvaud** [comment]
Hi,
A similar issue is open on the gym repo. But I think that box2d is only used for lunarlander environment so you should be able to use MuZero without box2d, (just remove it from requirements.txt).

**Comment by assist726** [comment]
@gWOLF3 I ran into the same issue too, If you do want to have box2d can just run `brew install swig` beforehand and the error goes away

**Comment by werner-duvaud** [comment]
Thanks
Should be fixed with e33d2548c6af4253d319dfcbac99fef88dcad75d
